### PR TITLE
Centos/RHEL installation guide update

### PR DESCRIPTION
**Pull request by mjmaenpaa**
Updated Centos 6/RHEL 6 installation guide for GitLab version 8.9

Still missing from guide:

- SELinux rules for Logrotate
- SELinux rules for Nginx

### CODE REVIEW

File: install/centos/README.md [hunk 1]
```diff
@@ -89,52 +90,45 @@ Now install the `epel-release-6-8.noarch` package, which will enable EPEL reposi
...
 
-    yum-config-manager --enable epel --enable PUIAS_6_computational
+    yum-config-manager --enable epel --enable remi-safe
 
 ### Install the required tools for GitLab
 
     yum -y update
     yum -y groupinstall 'Development Tools'
-    yum -y install readline readline-devel ncurses-devel gdbm-devel glibc-devel tcl-devel openssl-devel curl-devel expat-devel db4-devel byacc sqlite-devel libyaml libyaml-devel libffi libffi-devel libxml2 libxml2-devel libxslt libxslt-devel libicu libicu-devel system-config-firewall-tui redis sudo wget crontabs logwatch logrotate perl-Time-HiRes git cmake libcom_err-devel.i686 libcom_err-devel.x86_64 nodejs
+    yum -y install readline readline-devel ncurses-devel gdbm-devel glibc-devel tcl-devel openssl-devel curl-devel expat-devel db4-devel byacc sqlite-devel libyaml libyaml-devel libffi libffi-devel libxml2 libxml2-devel libxslt libxslt-devel libicu libicu-devel system-config-firewall-tui redis sudo wget crontabs logwatch logrotate perl-Time-HiRes git cmake libcom_err-devel.i686 libcom_err-devel.x86_64 nodejs python-docutils
```
Comment (axilleas) [inline install/centos/README.md]: Why do we need this package?

File: install/centos/README.md [hunk 2]
```diff
@@ -145,18 +139,6 @@ add the rhel6 optional packages repo to your server to get those packages:
...
-If you are familiar with vim set it as default editor with the commands below.
-If you are not familiar with vim please skip this and keep using the default editor.
-
-    # Install vim and set as default editor
-    yum -y install vim-enhanced
-    update-alternatives --set editor /usr/bin/vim.basic
-
-    # For reStructuredText markup language support, install required package:
-    yum -y install python-docutils
-
```
Comment (axilleas) [inline install/centos/README.md]: Why was this section removed?

File: install/centos/README.md [hunk 3]
```diff
@@ -226,27 +212,36 @@ Remove any other Ruby build if it is still present:
 Download Ruby and compile it:
 
     mkdir /tmp/ruby && cd /tmp/ruby
-    curl --progress ftp://ftp.ruby-lang.org/pub/ruby/2.1/ruby-2.1.2.tar.gz | tar xz
-    cd ruby-2.1.2
+    curl --progress ftp://ftp.ruby-lang.org/pub/ruby/2.1/ruby-2.1.10.tar.gz | tar xz
+    cd ruby-2.1.10
```
Comment (axilleas) [inline install/centos/README.md]: The latest 2.1 is 2.1.9, from https://cache.ruby-lang.org/pub/ruby/2.1/

File: install/centos/README.md [hunk 4]
```diff
@@ -226,27 +212,36 @@ Remove any other Ruby build if it is still present:
...
     ./configure --disable-install-rdoc
     make
     make prefix=/usr/local install
 
 Install the Bundler Gem:
 
-    gem install bundler --no-doc
+    gem install bundler --no-ri --no-rdoc
```
Comment (axilleas) [inline install/centos/README.md]: Actually `--no-doc` equals to `--no-ri --no-rdoc`.

File: install/centos/README.md [hunk 5]
```diff
@@ -336,6 +323,20 @@ If you see the following:
...
+    WHERE name = 'pg_trgm'
+    AND installed_version IS NOT NULL;
+
+If the extension is enabled this will produce the following output:
+
+    enabled
+    ---------
+     t
+     (1 row)
+
```
Comment (axilleas) [inline install/centos/README.md]: :+1: for adding this!

File: install/centos/README.md [hunk 6]
```diff
@@ -380,7 +387,7 @@ Create the GitLab production database:
 
 Grant the GitLab user necessary permissions on the table:
 
-    GRANT SELECT, LOCK TABLES, INSERT, UPDATE, DELETE, CREATE, DROP, INDEX, ALTER ON `gitlabhq_production`.* TO 'git'@'localhost';
+    GRANT SELECT, INSERT, UPDATE, DELETE, CREATE, CREATE TEMPORARY TABLES, DROP, INDEX, ALTER, LOCK TABLES ON `gitlabhq_production`.* TO 'git'@'localhost';
```
Comment (axilleas) [inline install/centos/README.md]: We need one more permission: `REFERENCES`, see https://gitlab.com/gitlab-org/gitlab-ce/merge_requests/5079

File: install/centos/README.md [hunk 7]
```diff
@@ -637,22 +716,37 @@ Validate your `gitlab` or `gitlab-ssl` Nginx config file with the following comm
...
+
+    service nginx restart
+
 ### Apache
 
 Httpd can be configured with or without SSL support.  Please choose appropriate commands in next steps.
 
+#### GitLab-Workhorse
+
+Apache installation requires changes to gitlab-workhorse configuration. Change `gitlab_workhorse_options` in `/etc/default/gitlab` to following.
```
Comment (axilleas) [inline install/centos/README.md]: to **the** following. Also, add a colon at the end.

**Comment by axilleas** [comment]
@mjmaenpaa the changes look awesome! Thanks a lot for this! I left some inline comments.

File: install/centos/README.md (hunk 1 above)
Comment (mjmaenpaa) [inline install/centos/README.md]: I used [Ubuntu guide](https://github.com/gitlabhq/gitlabhq/blob/master/doc/install/installation.md) as a base for updating this one and docutils and rst are not marked there as optional.

File: install/centos/README.md (hunk 4 above)
Comment (mjmaenpaa) [inline install/centos/README.md]: I know, but Ubuntu guide uses --no-ri --no-rdoc and I wanted to be as close to possible to that one. However Ubuntu guide probably should be updated to use --no-doc.

Comment (axilleas) [inline install/centos/README.md]: True, I'll see to change that.

**Comment by axilleas** [comment]
@mjmaenpaa Thank you!

**Pull request merged**